### ISSUE TITLE
Codes for training with ResNet18

### ISSUE THREAD

**Issue by ChunyunShen** [PRIS-CV/Mutual-Channel-Loss#8]
Hi authors:
    Could you release codes for training with ResNet18 (trained from scratch)? I find it is hard to obtain the reported scores (45.7 with CE loss) training with the hyperparameters provided in your paper. Maybe I miss some critical issues.

**Comment by dongliangchang** [comment]
all you need to do is to change the backbone in CUB-200-2011.py, and change the output channel of the backbone to 600.

**Comment by ChunyunShen** [comment]
Hi @dongliangchang 
According to your advise, I have trained ResNet18 from scratch based on this repo. The performance has improved! However the results I obtained (29.89 with CE loss, 51.97 with MC loss) are still much worse than reported in the paper (45.70 with CE loss, 59.41 with MC loss). 
Maybe the hyperparameters are not optimal... I am very confused.

**Comment by dongliangchang** [comment]
I will release the code.

**Comment by zhangchuanyi96** [comment]
> I will release the related codes within one month. Thanks for your attention.

Hi, can you please release the related codes? I am very interested in your research and need the codes for researching.

**Comment by dongliangchang** [comment]
Sorry for this. As I need to pass the Duolingo English Test this month, I don't have enough time to update this code. But the only difference between  the MC_loss with VGG16 and the  MC_loss with ResNet18 is the backbone, just change it for test. Maybe you can refer to this code (https://github.com/Kurumi233/Mutual-Channel-Loss).

**After I  pass the Duolingo  English Test, I will update this code.**

**Comment by zhangchuanyi96** [comment]
> Sorry for this. As I need to pass the Duolingo English Test this month, I don't have enough time to update this code. But the only difference between the MC_loss with VGG16 and the MC_loss with ResNet18 is the backbone, just change it for test. Maybe you can refer to this code (https://github.com/Kurumi233/Mutual-Channel-Loss).
> 
> **After I pass the Duolingo English Test, I will update this code.**
Great! Thank you for your help. I Sincerely wish you pass the Duolingo English Test.

**Comment by zhangchuanyi96** [comment]
> Sorry for this. As I need to pass the Duolingo English Test this month, I don't have enough time to update this code. But the only difference between the MC_loss with VGG16 and the MC_loss with ResNet18 is the backbone, just change it for test. Maybe you can refer to this code (https://github.com/Kurumi233/Mutual-Channel-Loss).
> 
> **After I pass the Duolingo English Test, I will update this code.**

Great! Thank you for your help. I Sincerely wish you pass the Duolingo English Test.

**Comment by dongliangchang** [comment]
@ChunyunShen @zhangchuanyi96 Hi,  I have update the related code. The results of this code is 59.66 ± 0.54 (59.41, 59.16, 60.41).
